### ISSUE TITLE
Webcam settings are being applied without confirmation

### ISSUE THREAD

**Issue by antonbsa** [bigbluebutton/bigbluebutton#14113]
Currently, if a user changes webcam settings (such as virtual background) while already sharing it, the settings will be applied automatically - right after clicking one of the options. 
Question: shouldn't this be applied only after a confirmation action - like clicking on the 'X'? It would be the same behavior as the connection status.

**Screenshots**
blur:
![blur](https://user-images.githubusercontent.com/56703993/149776949-cbc6929f-51b7-4096-9ba0-2205f0fe4823.png)

home:
![home](https://user-images.githubusercontent.com/56703993/149776986-75996c91-e5d6-40b7-8161-f73dc1231337.png)

**Comment by prlanzarin** [comment]
That's expected behavior/by design due to technical concerns, see: https://github.com/bigbluebutton/bigbluebutton/pull/13818#issuecomment-985048994

**Comment by prlanzarin** [comment]
Also: video-preview UI's not ideal and that's something folks already have in mind internally.
See https://github.com/bigbluebutton/bigbluebutton/issues/12836, https://github.com/bigbluebutton/bigbluebutton/issues/12945

The fact that the UI/UX is not _that_ clear will be mapped into an issue once somebody has a spec for a better video-preview approach.

**Comment by prlanzarin** [comment]
In case there's no reply I'll probably close this as a question/wontfix issue. This needs to be re-spec'ed as an overall video-preview rework (as mentioned in the previous comment); there's no workaround to be applied right now as far as this is concerned.

**Comment by antobinary** [comment]
^ Agreed. I fell for it too when testing some work in the area. It is just a different pattern.
